plugins/topology/README.md

```diff
@@ -145,6 +145,38 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
     app.openshift.io/vcs-ref: <GIT_REPO_BRANCH>
   ```
 
+  > Note: If Red Hat OpenShift Dev Spaces is [installed and configured](https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.7/html/administration_guide/installing-devspaces) and git URL annotations are also added to the workload YAML file, then clicking on the edit code decorator redirects you to the Red Hat OpenShift Dev Spaces instance.
+
+  > Note: When you deploy your application using the OCP git import flow, then you do not need to add the labels as import flow add the labels to the workload YAML file. Otherwise, you would need to add the labels to the workload YAML file manually.
```

Comment:
```suggestion
  > Note: When you deploy your application using the OCP git import flow, then you do not need to add the labels as import flow to the workload YAML file. Otherwise, you would need to add the labels to the workload YAML file manually.
```